app/components/wallet/trezorConnect/AboutDialog.js

```diff
@@ -0,0 +1,186 @@
+    description: 'First Prerequisite on the Connect to Trezor Hardware Wallet dialog.'
+  },
+  aboutPrerequisite1Part2LinkText: {
+    id: 'wallet.trezor.dialog.trezor.step.about.prerequisite.1.part2.link.text',
+    defaultMessage: '!!!Trezor Model T with version 2.0.8',
+    description: 'First Prerequisite on the Connect to Trezor Hardware Wallet dialog.'
+  },
+  aboutPrerequisite1Part3: {
+    id: 'wallet.trezor.dialog.trezor.step.about.prerequisite.1.part3',
+    defaultMessage: '!!!or later',
+    description: 'First Prerequisite on the Connect to Trezor Hardware Wallet dialog.'
+  },
+  aboutPrerequisite2: {
+    id: 'wallet.trezor.dialog.trezor.step.about.prerequisite.2',
+    defaultMessage: '!!!Trezor device must be pre-initialized',
+    description: 'Second Prerequisite on the Connect to Trezor Hardware Wallet dialog.'
+  },
+  aboutPrerequisite3: {
+    id: 'wallet.trezor.dialog.trezor.step.about.prerequisite.3',
+    defaultMessage: '!!!The computer needs to be connected to the Internet throughout the process',
+    description: 'Third Prerequisite on the Connect to Trezor Hardware Wallet dialog.'
+  },
+  aboutPrerequisite4: {
+    id: 'wallet.trezor.dialog.trezor.step.about.prerequisite.4',
+    defaultMessage: '!!!Only one Trezor device can be connected to the computer at any time',
+    description: 'Fourth Prerequisite on the Connect to Trezor Hardware Wallet dialog.'
+  },
+  aboutPrerequisite5: {
+    id: 'wallet.trezor.dialog.trezor.step.about.prerequisite.5',
+    defaultMessage: '!!!Trezor device screen must be unlocked',
+    description: 'Fifth Prerequisite on the Connect to Trezor Hardware Wallet dialog.'
+  },
+  aboutPrerequisite6: {
+    id: 'wallet.trezor.dialog.trezor.step.about.prerequisite.6',
+    defaultMessage: '!!!Trezor device must remain connected to the computer throughout the process',
+    description: 'Sixth Prerequisite on the Connect to Trezor Hardware Wallet dialog.'
+  },
+  nextButtonLabel: {
+    id: 'wallet.trezor.dialog.trezor.next.button.label',
+    defaultMessage: '!!!Next',
+    description: 'Label for the "Next" button on the Connect to Trezor Hardware Wallet dialog.'
+  },
+});
+
+messages.fieldIsRequired = globalMessages.fieldIsRequired;
+
+type Props = {
+  progressInfo: ProgressInfo,
+  error: ?LocalizableError,
+  submit: Function,
+  cancel: Function,
+};
+
+@observer
+export default class AboutDialog extends Component<Props> {
+
+  static contextTypes = {
+    intl: intlShape.isRequired
+  };
+
+  render() {
+    const { intl } = this.context;
+
+    // introBlock
+    const introBlock = (
+      <div className={styles.headerBlock}>
+        <span>{intl.formatMessage(messages.aboutIntroTextLine1)}</span><br />
+        <span>{intl.formatMessage(messages.aboutIntroTextLine2)}</span><br />
+        <span>{intl.formatMessage(messages.aboutIntroTextLine3)}</span><br />
+      </div>);
+
+    // middleBlock
```

Comment:
I would delete this comment because it says the same as the name of the variable so it doesn’t add any new information